portfolio.html

```diff
@@ -37,6 +37,128 @@ <h1 class="item2">I'm a software developer</h1>
+      <div class="a1">css</div>
+      <div class="a1">JavScript</div>
+      <div class="a1">html</div>          
+    </div>
+    <div class="b_div"><button class="button">See Project</button></div>
+   </div>
+
+    <div class="card-container">
+    <div class="card-1">
+      <img src="Img Plaholder@1x.png" alt="project image">
+    </div>
+      
+    <div class="card-2">
+      <div class="title"><h3>Multi-Post Stories Gain+Glory</h3></div>
+      <div class="abilites">
+      <div class="a1">Ruby on rails</div>
+      <div class="a1">css</div>
+      <div class="a1">JavScript</div>
+      <div class="a1">html</div>          
+    </div>
+    <div class="b_div"><button class="button">See Project</button></div>
+   </div>
+
+   <div class="card-container">
+    <div class="card-1">
+      <img src="Img Plaholder@1x.png" alt="project image">
+    </div>
+      
+    <div class="card-2">
+      <div class="title"><h3>Multi-Post Stories Gain+Glory</h3></div>
+      <div class="abilites">
+      <div class="a1">Ruby on rails</div>
+      <div class="a1">css</div>
+      <div class="a1">JavScript</div>
+      <div class="a1">html</div>          
+      </div>
+      <div class="b_div"><button class="button">See Project</button></div>
+    </div>
+
+   <div class="card-container">
+    <div class="card-1">
+      <img src="Img Plaholder@1x.png" alt="project image">
+    </div>
+
+    <div class="card-2">
+      <div class="title"><h3>Multi-Post Stories Gain+Glory</h3></div>
+      <div class="abilites">
+      <div class="a1">Ruby on rails</div>
+      <div class="a1">css</div>
+      <div class="a1">JavScript</div>
+      <div class="a1">html</div>          
+    </div>
+    <div class="b_div"><button class="button">See Project</button></div>
+
+   </div>
+
+   <div class="card-container">
+    <div class="card-1">
+      <img src="Img Plaholder@1x.png" alt="project image">
+    </div>
+      
+    <div class="card-2">
+      <div class="title"><h3>Multi-Post Stories Gain+Glory</h3></div>
+      <div class="abilites">
+      <div class="a1">Ruby on rails</div>
+      <div class="a1">css</div>
+      <div class="a1">JavScript</div>
+      <div class="a1">html</div>          
+    </div>
+    <div class="b_div"><button class="button">See Project</button></div>
+   </div>
+
+   <div class="card-container">
+      <div class="card-1">
+        <img src="Img Plaholder@1x.png" alt="project image">
+      </div>
+      
+     <div class="card-2">
+       <div class="title"><h3>Multi-Post Stories Gain+Glory</h3></div>
+         <div class="abilites">
+         <div class="a1">Ruby on rails</div>
+         <div class="a1">css</div>
+         <div class="a1">JavScript</div>
+         <div class="a1">html</div>          
+     </div>
+      <div class="b_div"><button class="button">See Project</button></div>
+    </div>
+
+   </section>
+
+    <section class="about_me_container">
+      <h1 class="about_me1">About me</h1>
```

Comment:
- [ ] Lastly, the spacing between the `works-section `and the `about-me section `isn't similar to the Figma design. It is smaller compared to the Figma design.  Please try to stick to the design guidelines as much as possible so you can fix this as well. You can increase the `margin-top `of the` about-me `section to fix this :wink: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/219852290-8c4bbc62-f2ae-4453-91ab-de95b7399011.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/219852322-ed65e5dd-67fe-4dee-8d6b-9d80f77e405b.png)